public/components/settings/api/api-table.js

```diff
@@ -122,16 +127,34 @@ export const ApiTable = compose(withErrorBoundary, withReduxProvider)(class ApiT
         entries[idx].status = 'online';
       } catch (error) {
         const code = ((error || {}).data || {}).code;
-        const downReason = typeof error === 'string' ? error :
-        (error || {}).message || ((error || {}).data || {}).message || 'Wazuh is not reachable';
+        const downReason =
+          typeof error === 'string'
+            ? error
+            : (error || {}).message ||
+              ((error || {}).data || {}).message ||
+              'Wazuh is not reachable';
         const status = code === 3099 ? 'down' : 'unknown';
-        entries[idx].status = { status, downReason }; 
+        entries[idx].status = { status, downReason };
+        throw error;
```

Comment:
I think this is a expected error and it should not be trowed, the component is handling the error

Reply:
Yes in this case the component catches these errors and show them on a `ToolTip` like this:
![image](https://user-images.githubusercontent.com/11060558/123160452-9d7cd280-d444-11eb-8759-44c263191013.png)
I think could be better to log them on our file `.log` too.